File: 1635-NumberOfGoodPairs/1635-NumberOfGoodPairs.java
// Last updated: 9/3/2025, 10:47:35 AM
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int good=0;
        for(int i =0;i<nums.length-1;i++){
            for(int j =i+1;j<nums.length;j++){
                if(nums[i]==nums[j]) good++;
            }
        }
        return good;
    }
}

File: 1635-NumberOfGoodPairs/1635-NumberOfGoodPairs.py
# Last updated: 9/3/2025, 10:47:33 AM
class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        good=0
        for i in range (0,len(nums)):
            for j in range (i+1,len(nums)):
                if nums[i]==nums[j]:
                    good+=1
        return good
        